mentors/javascript/a.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "github_username": "kytrinyx",
```

```suggestion
    "github_username": "abredi",
```
You need to put your GitHub user name here......